src/ml/neural_net/style_transfer/mps_style_transfer_decoding_node.mm

```diff
@@ -71,4 +74,34 @@ - (MPSNNImageNode *) backwardPass:(MPSNNImageNode *) inputNode {
   return [upsampleGrad resultImage];
 }
 
+- (void)setLearningRate:(float)lr {
+  [[_conv weights] setLearningRate:lr];
+  [[_instNorm weights] setLearningRate:lr];
+}
+
+- (NSDictionary<NSString *, NSData *> *)exportWeights {
+  NSMutableDictionary<NSString *, NSData *> *weights;
+
+  NSUInteger convWeightSize = (NSUInteger)([[_conv weights] weight_size] * sizeof(float));
+
+  NSMutableData* convDataWeight = [NSMutableData dataWithCapacity:convWeightSize];
```

Comment:
Use `dataWithLength:` to ensure the buffer is actually allocated

Reply:
Proper. Changed the method.